source/GameMode.cpp

```diff
@@ -60,16 +67,20 @@ bool GameMode::init(const std::shared_ptr<cugl::AssetManager>& assets) {
 
 	input.init();
 	net.initHost();
+	CULog("%s", net.getRoomID());
```

Comment:
You need to turn the `std::string` that `mib` returns into a c string before you can pass it to `CULog`. Try to `c_str()` method.

Reply:
deleted that line anyways